libs/cart/src/drivers/magento/cart-shipping-information.service.spec.ts

```diff
@@ -212,11 +224,40 @@ describe('Driver | Magento | Cart | CartShippingInformationService', () => {
         done();
       });
 
-      const op = controller.expectOne(addTypenameToDocument(setSelectedShippingMethod([daffMagentoNoopCartFragment])));
+      const setSelectedShippingMethodOp = controller.expectOne(addTypenameToDocument(setSelectedShippingMethod([daffMagentoNoopCartFragment])));
 
-      op.flush({
+      setSelectedShippingMethodOp.flush({
         data: mockSetSelectedShippingMethodResponse
       });
+
+      // set timeout because the requests here are made in serial
+      setTimeout(() => {
+        const listShippingMethodsOp = controller.expectOne(addTypenameToDocument(listShippingMethods([daffMagentoNoopCartFragment])));
+
+        listShippingMethodsOp.flush({
+          data: mockListCartShippingMethodsResponse
+        });
+      })
+    });
+
+    it('should manually refetch the shipping methods and force cache bypass', () => {
```

Comment:
This test doesn't actually register that there are any expectations, so it counts as a skipped test. Probably because of setTimeout

Reply:
Its because the actual checking happens in `afterEach`. Unfortunately there's not really a good way to check that the request bypasses the cache so I removed that spec since its not doing anything more than the other one.

Reply:
that's fine with me